pinot-core/src/main/java/org/apache/pinot/core/accounting/CPUMemThreadLevelAccountingObjects.java

```diff
@@ -154,10 +158,15 @@ public TaskType getTaskType() {
       return _taskType;
     }
 
+
+    public String getWorkloadName() {
+      return _workloadName == null ? CommonConstants.Accounting.DEFAULT_WORKLOAD_NAME : _workloadName;
```

Comment:
Can the default workload name be forced for a query when a runner/worker is setup ? That will remove this null-check.

Reply:
I read `QueryOptionsUtil` after this file. default workload name is chosen if not set. So is this null check required ?